GNUmakefile

```diff
@@ -147,15 +147,10 @@ else
 	@printf "\e[36mVim mode installed.\e[39m\n"
 endif
 
-# Install for the emacs mode (system-wide).
 .PHONY: install_emacs
 install_emacs:
-ifeq ($(EMACS),)
-	@printf "\e[36mWill not install emacs mode.\e[39m\n"
-else
-	@cd editors/emacs && $(MAKE) install
-	@printf "\e[36mEmacs mode installed.\e[39m\n"
-endif
+	@printf "\e[36mAdd (load \"$(EMACSDIR)/lambdapi-site-file\") to \
+your Emacs configuration file.\e[39m\n"
```

Comment:
In that case, I guess that the `install_emacs` target should depend on the `install target` right? Otherwise it only shows the message.

On the other end, I'm wondering if this is even useful now. We could very well remove every mention to emacs in the `Makefile`.